modules/rhel-compute-requirements.adoc

```diff
@@ -51,6 +51,10 @@ be configured according to its
 link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/index.html[storage guidelines]
 and the `disk.enableUUID=true` attribute must be set.
 
+* Each system should be able to access the cluster's API
+endpoints using a DNS resolvable hostname. Any network security access control in place should allow the system access to the
+cluster's API service endpoints. 
```

Comment:
Does the first sentence cover this entirely if it's changed to:

"Each system must be able to access the cluster's API endpoints."

Is it worth calling out access controls specifically after that?

Reply:
Yes. We have to call out the network security problem because that is actually the root cause of the original bug. They did not set the network security correctly.